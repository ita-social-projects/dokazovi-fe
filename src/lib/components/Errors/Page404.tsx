import React from 'react';
import { Typography } from '@material-ui/core';
import { PageTitle } from '../Pages/PageTitle';

const Page404: React.FC = () => {
  return (
    <>
      <PageTitle title="Помилка 404" />
      <Typography align="center" variant="h1">
        404 Page not found
      </Typography>
    </>
  );
};

export default Page404;
