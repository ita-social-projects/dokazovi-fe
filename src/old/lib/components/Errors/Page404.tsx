import React, { useEffect } from 'react';
import { Typography } from '@material-ui/core';
import { PageTitle } from '../Pages/PageTitle';
import { setGALocation } from '../../../../utilities/setGALocation';

const Page404: React.FC = () => {
  useEffect(() => {
    setGALocation(window);
  }, []);

  return (
    <>
      <PageTitle title="Помилка 404" />
      <Typography align="center" variant="h1">
        404 Page not found
      </Typography>
    </>
  );
};

export default Page404;
