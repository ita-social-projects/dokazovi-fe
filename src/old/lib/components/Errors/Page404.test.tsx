import React from 'react';
import { render } from '@testing-library/react';
import Page404 from './Page404';

describe('Page404', () => {
  it('render component and match snapshot', () => {
    const { asFragment } = render(<Page404 />);
    expect(asFragment()).toMatchSnapshot();
  });
});
