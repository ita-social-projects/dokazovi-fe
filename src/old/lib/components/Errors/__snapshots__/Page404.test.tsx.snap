// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page404 render component and match snapshot 1`] = `
<DocumentFragment>
  <div
    style="margin: auto;"
  >
    <h1
      class="MuiTypography-root MuiTypography-h1 MuiTypography-alignCenter"
    >
      На жаль, таку сторінку не знайдено.
    </h1>
  </div>
</DocumentFragment>
`;
